src/components/TrapFocus/TrapFocus.tsx

```diff
@@ -68,25 +78,25 @@ export class TrapFocus extends React.PureComponent<TrapFocusProps, State> {
   };
 
   private handleBlur = (event: FocusEvent) => {
-    const {relatedTarget} = event;
+    const {relatedTarget: currentTarget} = event;
```

Comment:
you probably just want to keep this as `relatedTarget`. Renaming it to currentTarget is a bit confusing since it isn't the curentTarget.

Reply:
➕ 1️⃣